app/controllers/users_controller.rb

```diff
@@ -112,6 +111,9 @@ def show
     @phenotype_comment_replies.sort! { |b,a| a.created_at <=> b.created_at }
     @paginated_phenotype_replies = @phenotype_comment_replies
 
+    # get the last 30 updated SNPs for this user
+    @last_updated_snps = @user.snps.order('updated_at desc').limit(30) unless @user.snps.nil?
```

Comment:
`@user.snps.nil?` will always be false, since `@user.snps` returns an empty array, if there are no Snps.